src/R/Editor/Impl/Completion/Providers/PackageFunctionCompletionProvider.cs

```diff
@@ -60,8 +60,12 @@ public class FunctionCompletionProvider : IRCompletionListProvider {
                 IEnumerable<INamedItemInfo> functions = pkg.Functions;
                 if (functions != null) {
                     foreach (INamedItemInfo function in functions) {
-                        bool? isSnippet = infoSource?.IsSnippet(function.Name);
-                        if (!isSnippet.HasValue || !isSnippet.Value) {
+                        bool isSnippet = false;
+                        // Snippets are suppressed if user typed namespace
+                        if (!context.IsInNameSpace() && infoSource != null) {
```

Comment:
Also, we don't show anything after $ except variable members (if any) - i.e. only one completion list provider is active, so no snippets, keywords or functions appear anyway. ```::``` is pretty much the only case when we show functions.